lib/shared/widgets/index.dart

```diff
@@ -0,0 +1,2 @@
+import './stateful_widget.dart';
+import './stateless_widget.dart';
```

Comment:
@27vinayvyas Same no need to create these files. Delete them.

Reply:
@yashwp The widgets and models(#4 and #5) issued are not useful then.

Reply:
Yeah... You can remove them.

Reply:
@27vinayvyas